flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/scala/DataStreamConversions.scala

```diff
@@ -59,5 +59,38 @@ class DataStreamConversions[T](dataStream: DataStream[T], inputType: TypeInforma
     }
   }
 
+  /**
+    * Converts the [[DataStream]] with upsert messages into a [[Table]] with keys.
+    *
+    * The message will be encoded as [[Tuple2]]. The first field is a [[Boolean]] flag, the second
```

Comment:
The messages in the table will be encoded as `Tuple2`? Or the incoming messages from the source `DataStream` are expected to be encoded as `Tuple2`?

Reply:
The incoming messages from the source `DataStream` are expected to be encoded as `Tuple2`. I will change the description according to your suggestion. Thank you.